pom.xml

```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>me.totalfreedom</groupId>
+    <groupId>com.github.AtlasMediaGroup</groupId>
     <artifactId>TFGuilds</artifactId>
-    <version>2021.06-RC3</version>
+    <version>master-4ec00d977c-1</version>

```

Comment:
Could you change this back though, this should be our actual release version, not a commit ID from the master branch.

Reply:
changed